docs/topics/api/v4_frozen/accounts.rst

```diff
@@ -58,7 +58,6 @@ If you have `Users:Edit` permission you will see these extra fields for all user
 
     :>json boolean deleted: Is the account deleted.
     :>json string|null display_name: The name chosen by the user.
-    :>json string|null reviewer_name: The reviewer name chosen by the user, if set. Only available for users with any kind of reviewer permission.
```

Comment:
MMM normally since it's frozen I'd say this needs a shim for backwards-compatibility... But given it was only restricted to those specific users, maybe that's fine...